tools/ci_build/github/azure-pipelines/c-api-packaging-pipelines.yml

```diff
@@ -10,22 +10,22 @@ jobs:
     - task: Docker@2
       displayName: login
       inputs:
-        containerRegistry: onnxruntimeregistry
+        containerRegistry: onnxruntimebuildcache
```

Comment:
this update is ok in the short term, but can we move to using get_docker_image.py at some point so the images can be built and cached automatically?

Reply:
Sure. That would be great! This PR is meant to solve the current problem, to get a clean state of onnxruntimeregistry.

Reply:
ok, that sounds good. just fyi, we shouldn't rely on images persisting in onnxruntimebuildcache after i enable the automatic cache clean up (#5843).